Bottom: f56818753863d40003f0da78fadee5294c7bd144
Top:    9447ceeca50a61971e7c745375b6e45d6b003deb
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-25 18:00:19 -0400

Refresh of phytec

---

diff --git a/sound/soc/fsl/pcm030-audio-fabric.c b/sound/soc/fsl/pcm030-audio-fabric.c
index 2c426d5..8766f7a 100644
--- a/sound/soc/fsl/pcm030-audio-fabric.c
+++ b/sound/soc/fsl/pcm030-audio-fabric.c
@@ -81,12 +81,7 @@ static __init int pcm030_fabric_init(void)
 	return 0;
 }
 
-static __exit void pcm030_fabric_exit(void)
-{
-}
-
 module_init(pcm030_fabric_init);
-module_exit(pcm030_fabric_exit);
 
 
 MODULE_AUTHOR("Jon Smirl <jonsmirl@gmail.com>");
